beta-src/src/components/ui/WDPositionContainer.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import { useMemo } from "react";
+
+export enum Position {
+  BOTTOM_LEFT = "bottom-left",
+  BOTTOM_RIGHT = "bottom-right",
+  TOP_LEFT = "top-left",
+  TOP_RIGHT = "top-right",
+}
+
+interface WDPositionContainerProps {
+  children: React.ReactNode;
+  position?: Position;
+}
+
+const MOBILE_DISTANCE = 16;
+const TABLET_UP_DISTANCE = 24;
+const Z_INDEX = 2;
+
+const responsiveDistance = {
+  mobile: MOBILE_DISTANCE,
+  tablet: TABLET_UP_DISTANCE,
+  desktop: TABLET_UP_DISTANCE,
+} as const;
```

Comment:
Wondering the same here.  If this is specific to this component then it's okay.

Reply:
yes, this is specific to this component. if later we discover there are other responsive distances that are the same, we might add it as a theme. for now, i think its ok to have this here.